src/main/cheat_code.cpp

```diff
@@ -2219,8 +2325,8 @@ static void processCheats()
 
     if(cheated)
     {
-        pLogDebug("Cheating detected!!! [%s]\n", oldString.c_str());
-        Cheater = true;
+        pLogDebug("Cheating detected! The game can still save though. [%s]\n", oldString.c_str());
+        Cheater = false;
```

Comment:
You shall NOT remove the "no-save" punishment for cheating. Yet. Or add the `compat.ini` option that declares this behaviour (for now, it must be DISABLED by default), it's significantly important.